cmd/zbpack/main_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+)
+
+var binName = "zbpack"
+
+func TestMain(m *testing.M) {
+	fmt.Println("Running build...")
+
+	if runtime.GOOS == "windows" {
+		binName += ".exe"
+	}
+
+	build := exec.Command("go", "build", "-o", binName)
```

Comment:
Output it to a temporary directory like `/tmp`. Don't place your test executable file in the working directory, which may introduce an unexpected result.